src/assets/scripts/client/parsers/aircraftCommandParser/argumentValidators.js

```diff
@@ -168,6 +168,23 @@ export const fixValidator = (args = []) => {
     }
 };
 
+/**
+ * Returns true if the value is exactly a 3 digit decimal number
+ *
+ * @function isValidCourseString
+ * @param value {string}
+ * @returns {boolean}
+ */
+export const isValidCourseString = (value) => {
+    // Can't rely on parseInt/convertStringToNumber as it'll parse 1min => 1
+    if (!/^[0-9]{3}$/.test(value)) {
+        return false;
+    }
+
+    const course = convertStringToNumber(value);
+    return course >= 0 && course <= 360;
```

Comment:
Not sure if it matters in this instance, but `0` for a heading is never right to display to the user. If we're converting between radians and degrees we may end up here which is fine. Just want to verify/reiterate that only 001 through 360 should be what the user sees anywhere.

Reply:
Yup, all the readbacks are normalised to 001 -> 360. I left this as allowing 000, as it's for validating user input, much in the same way `fh 000` is an acceptable command

Reply:
`fh 0` or `fh 000` is not an acceptable command, it should definitely assume that to be a typo and reject a zero-heading.

That said, if it's what it was already doing before these changes, you don't have to go change that if you don't want to-- only if it's a new behavior.

Reply:
Ah, `fh 000` may not be an acceptable command, but it is currently accepted by the sim as a valid command.

I'm happy change it so 000 isn't valid for the `hold` command if you like. I imagine you'd prefer the runHeading behaviour is fixed outside of this PR.